src/pages/PullRequestPage/hooks/usePullPageData.tsx

```diff
@@ -68,7 +68,6 @@ query PullPageData($owner: String!, $repo: String!, $pullId: Int!) {
     repository(name: $repo) {
       __typename
       ... on Repository {
```

Comment:
Confirming we no longer use private

Reply:
yep, i just missed it when i cleaned up the use of current user part of org